tools/build.js

```diff
@@ -51,9 +51,15 @@ const configureReactNative = () => {
   // cleanup old build
   const reactNativeFolder = './react-native';
   fs.removeSync(path.resolve(process.cwd(), `${reactNativeFolder}/dist`));
-  rnSnTablePackage.version = snTablePackage.version;
+  const nextVersion = rnSnTablePackage.version.split('.');
+  const minor = parseInt(nextVersion[nextVersion.length - 1], 10) + 1;
+  nextVersion[nextVersion.length - 1] = minor;
+  rnSnTablePackage.version = nextVersion.join('.');
```

Comment:
So this bumps the minor version every time you build it for react native? How would that work when you want to make a release? it is always a minor bump?

Reply:
We're still working on a better build pipeline.  I would like to use semantic release.

Reply:
ok, if this works for you now it is fine with me